src/components/Planters.js

```diff
@@ -230,6 +230,7 @@ const Planters = (props) => {
             buttons={[
               <IconButton onClick={handleFilterClick} key={1}>
```

Comment:
This would be better as a regular `Button` with a `startIcon`.
See https://material-ui.com/components/buttons/#buttons-with-icons-and-label.